application_configuration.md

```diff
@@ -106,12 +90,15 @@ gem 'devise-i18n'
 gem 'figaro'
 gem 'font-awesome-rails'
 gem 'goldiloader'
+gem 'http_accept_language'
+gem 'i18n-docs'
 gem 'jquery-rails'
 gem 'jquery-turbolinks'
 gem 'pg'
 gem 'rack-google-analytics'
 gem 'rails'
 gem 'rails-i18n'
+gem 'rails_real_favicon'
```

Comment:
ah, i thought it needs to be in the production env. actually, i looked at the code here https://github.com/RealFaviconGenerator/rails_real_favicon/blob/master/lib/generators/favicon_generator.rb and it seemed really simple and also well tested https://github.com/RealFaviconGenerator/rails_real_favicon/tree/master/test

let's also discuss this face to face :+1: